tools/indexer/example/src/main.rs

```diff
@@ -3,12 +3,61 @@ use std::io;
 
 use actix;
 
+use clap::Clap;
 use tokio::sync::mpsc;
 use tracing_subscriber::filter::LevelFilter;
 use tracing_subscriber::EnvFilter;
 
 use near_indexer;
 
+/// NEAR Indexer Example
+/// Watches for stream of blocks from the chain
+#[derive(Clap, Debug)]
+#[clap(version = "0.1", author = "Near Inc. <hello@nearprotocol.com>")]
+struct Opts {
+    /// Sets a custom config dir. Defaults to ~/.near/
+    #[clap(short, long, default_value = "~/.near/")]
```

Comment:
I wonder if that actually works as `~` is only resolved to the home folder on the shell level but if you explicitly try to open/create a folder `"~/.near"` is will treat it like a relative path with `~` being a literal naming of a folder, not the home folder.

Reply:
good catch, fixed